conu/utils/filesystem.py

```diff
@@ -49,8 +60,22 @@ def __init__(self, path, mode=None, facl_rules=None, selinux_context=None, selin
         self.selinux_range = selinux_range
         self.facl_rules = facl_rules
 
-        # if set to True, then the directory is created and set up
-        self._initialized = False
+        # os.chown wants int
+        if isinstance(user_owner, six.string_types):
+            try:
+                self.owner = pwd.getpwnam(user_owner)[2]
+            except KeyError as ex:
+                raise ConuException("User %r not found, error message: %r" % (user_owner, ex))
+        else:
+            self.owner = user_owner
+        if isinstance(group_owner, six.string_types):
+            try:
+                self.group = pwd.getpwnam(group_owner)[3]
+            except KeyError as ex:
+                raise ConuException("Group %r not found, error message: %r" % (group_owner, ex))
+        else:
+            self.group = group_owner
```

Comment:
Beautiful! :)

Reply:
I added this after I wrote tests -- would be so annoying to people.